roles/edpm_nova/defaults/main.yml

```diff
@@ -31,3 +31,14 @@ edpm_nova_compute_image: "quay.io/podified-antelope-centos9/openstack-nova-compu
 
 # certs
 edpm_nova_tls_certs_enabled: "{{ edpm_tls_certs_enabled | default(False) }}"
+# default list example
+edpm_nova_compute_services_to_stop:
```

Comment:
would use a name that signals that these are old tripleo services. What about `edpm_old_tripleo_compute_sevices`